examples/spawn-and-move/Scarb.toml

```
@@ -23,6 +20,5 @@ name = "example"
 rpc_url = "http://localhost:5050/"
 
 # Default account for katana with seed = 0
-account_address = "0x6162896d1d7ab204c7ccac6dd5f8e9e7c25ecd5ae4fcb4ad32e57786bb46e03"
+account_address = "0x517ececd29116499f4a1b64b094da79ba08dfd54a3edaa316134c41f8160973"
```

```suggestion
account_address = "0x6162896d1d7ab204c7ccac6dd5f8e9e7c25ecd5ae4fcb4ad32e57786bb46e03"
```
We should keep the new accounts instead. The one you're using if from the stable `v0.4.4`, but the idea is to adjust the example to the version we're currently building on main. :+1:

To check those addresses, you can build katana on main, this will give you the new ones.

This also makes me realize that sozo testing does not cover the usage of the `Scarb.toml`. @lambda-0x if you are around, this is something that @kariy already pointed out. We should use this `Scarb.toml` in a sozo migrate test to ensure we're using up to date Katana information.